examples_notebooks/community_contrib/couchbase/GraphRAG_with_Couchbase.ipynb

```diff
@@ -0,0 +1,656 @@
+{
+ "cells": [
+  {
```

Comment:
Also make example content similar to https://github.com/microsoft/graphrag/blob/de65413ec5c20094bc2e9fdf306ba175cbd9a792/examples_notebooks/local_search.ipynb

Reply:
have broken down the paraquet reading files into multiple cells as discussed